### DISCUSSION TITLE
Spline Mesh Component

### DISCUSSION THREAD

**Discussion by AaronWeis** [nxrighthere/UnrealCLR#260] [Help]
Hello,

I am currently working on a traffic simulation software for my studies. For the simulation of the roads I use splines, but I currently can't find a way to use spline mesh components. So my question: Is there a way to use Spline Mesh Components and if yes, how does it look like?
Thank you in advance and have a nice day,

Questus

**Comment by nxrighthere** [comment]
If you are looking for [`USplineMeshComponent`](https://docs.unrealengine.com/5.0/en-US/API/Runtime/Engine/Components/USplineMeshComponent/) then it's not exposed to the C# framework. It's on the [roadmap](https://github.com/users/nxrighthere/projects/5?fullscreen=true).
